kiwi/models/model.py

```diff
@@ -206,14 +207,19 @@ def get_mask(self, batch, output):
 
     @staticmethod
     def create_from_file(path):
-        model_dict = torch.load(
-            str(path), map_location=lambda storage, loc: storage
-        )
+
+        try:
+            model_dict = load_torch_file(path)
+        except FileNotFoundError:
+            # If no model is found
+            raise FileNotFoundError(
```

Comment:
Re-raising shouldn't be necessary. Do we lose too much on the user friendly error message?